### PR TITLE
Implement vcat and map for TimeArray

### PR DESCRIPTION
**Pull request by dourouc05**
Usual functions to work with arrays. Implemented `vcat` and not `hcat` or `cat`, because they sound more like `merge`.

### CODE REVIEW

**Comment by milktrader** [comment]
These look like reasonable adds, what do you think @GordStephen ?

File: src/combine.jl [hunk 1]
```diff
@@ -82,3 +82,41 @@ function collapse{T,N,D}(ta::TimeArray{T,N,D}, f::Function; period::Function=wee
...
+  # Concatenate the contents. 
+  timestamps = vcat([ta.timestamp for ta in TA]...)
+  values = vcat([ta.values for ta in TA]...)
+  return TimeArray(timestamps, values, TA[1].colnames, TA[1].meta)
+end
+
+# map ######################
+
+function map{T,N,D,A}(f::Function, ta::TimeArray{T,N,D,A})
+  timestamps = Array(typeof(ta.timestamp[1]), length(ta))
```
Comment (GordStephen) [inline src/combine.jl]: Can this just be `timestamps = similar(ta.timestamp)`?

**Comment by GordStephen** [comment]
Looks good to me!

**Comment by milktrader** [comment]
Ok, we can clean up the indentation later. Thanks @dourouc05

**Pull request merged**